src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -532,7 +532,7 @@ private async Task<ConnectBlocksResult> FullyValidateLockedAsync(ChainedHeader n
                 // In that case we try to load the blocks from store, if store is not present we disconnect all peers.
                 this.HandleMissingBlocksGap(currentTip);
 
-                var result = new ConnectBlocksResult(false);
+                var result = ConnectBlocksResult.Fail();
```

Comment:
Not sure this is actually clearer. It reads like you're making it fail, rather than expressing that "it's failed".

Reply:
Will rename to `Failed`